lifetiles-sequence/src/main/java/nl/tudelft/lifetiles/sequence/controller/SequenceController.java

```diff
@@ -125,6 +133,46 @@ private void registerShoutListeners() {
             updateVisible(visibleSet);
             shout(Message.FILTERED, "", visibleSet);
         });
+
+        listen(Message.OPENED,
+                (sender, subject, args) -> {
+                    if (!subject.equals("meta")) {
+                        return;
+                    }
+                    assert args[0] instanceof File;
+
+                    SequenceMetaParser parser = new SequenceMetaParser();
+                    try {
+                        parser.parse((File) args[0]);
+                        addMetaData(parser);
```

Comment:
A method should do only one thing. In this case, add metadata to sequence entries. Passing it the data it should add makes sense. Passing it a parser is odd, because the method is now responsible for two things: 'using' the parser, and adding the data.